### ISSUE TITLE
Split website and wiki

### ISSUE THREAD

**Issue by SuperQ** [noisebridge/infrastructure#167]
There was a proposal to split the website into two sites:
* One official static content site.
* The wiki.

This would bring us back to having useful wiki as a top-level thing. At the same time allowing for a more "professional" site for the organization.

**Comment by old-noisebridger** [comment]
Please review #171 and #172 and make sure the people who build the static site know HTML before moving ahead further with this proposal. Also what does professional even mean.

**Comment by marcidy** [comment]
Ignoring the comment above which is irrelevant to this issue, I believe this split is complete?

**Comment by marcidy** [comment]
@rizend @SuperQ sorry, didn't tag anyone, this is done right?

**Comment by SuperQ** [comment]
No, it's not done. We still have a static site on noisebridge.net. We wanted to move the static content to noisebridge.org and put the wiki home page back on noisebridge.net.

**Comment by marcidy** [comment]
Ah, ok, i misunderstood.  

The original requirement is "if an organization googles Noisebridge, they find a clean page" not "a static page somewhere".  I want to be sure we don't lose focus on that.

**Comment by mcint** [comment]
@rlew631 mentioned this goal to me. I want a CMS that or a way to quickly and easily update from just the web, for the .org site.

**Comment by SuperQ** [comment]
@mcint What about something like [Hugo](https://gohugo.io/)? We could have a `noisebridge.org` website repo. People can make quick and easy web updates via github. We can have a GitHub action to publish the hugo site.

**Comment by jof** [comment]
+1 for a static site generator like Hugo or Jekyll; a very hacker-friendly choice.

I will also mention Netlify and Vercel as some nice static site generation as-a-service platforms.

I think the Vercel free hobby or the Netlify free starter plans might work, but would be limited to a single admin on the web interface, but could allow for unlimited folks to push commits in git that trigger a build-and-deploy workflow.

**Comment by SuperQ** [comment]
Netlify is pretty great. Not sure if we would want to use their actual hosting platform since we want to keep the logs of users visiting the site relatively anonymous. We have a [standard web log filter](https://github.com/noisebridge/infrastructure/blob/1065719716b5b3f63a7d9dee05e0c6006973256d/group_vars/all/caddy.yml#L12-L22) that makes sure to not log too much.

**Comment by mcint** [comment]
> @mcint What about something like [Hugo](https://gohugo.io/)? We could have a `noisebridge.org` website repo. People can make quick and easy web updates via github. We can have a GitHub action to publish the hugo site.

I want more than just github users to modify it. Although version control also seems essential to me, which most likely means github or another database-backed system.
